common/lib/capa/capa/responsetypes.py

```diff
@@ -1931,6 +1937,22 @@ def get_score(self, student_answers):
                 if compare_with_tolerance(student_float, correct_float, expanded_tolerance):
                     is_correct = 'partially-correct'
 
+        # Compare with additional answers.
+        if is_correct == 'incorrect':
+            additional_answer_idx = 0
+            for additional_answer in self.additional_answers:
+                answer = self.get_staff_ans(additional_answer)
+                if compare_with_tolerance(
+                        student_float,
+                        answer,
+                        tolerance=float_info.epsilon,
+                        relative_tolerance=True
```

Comment:
I think we don't need to specify `tolerance` & `relative_tolerance` parameters because we're not supporting tolerance for additional answers.